src/Http/Resources/TopicExportResource.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace EscolaLms\Courses\Http\Resources;
+
+use EscolaLms\TopicTypes\Http\Resources\TopicType\Contacts\TopicTypeResourceContract;
+use EscolaLms\TopicTypes\Models\Contracts\TopicContentContract;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class TopicExportResource extends JsonResource
+{
+    /**
+     * @var array
+     *            All possible classes that can store content
+     */
+    private static array $contentClasses = [
+    ];
+
+    /**
+     * @param string $class fullname of a class that can be content
+     *
+     * @return array list of unique classes
+     */
+    public static function registerContentClass(string $modelClass, string $resourceClass): array
```

Comment:
Shouldn't there be some Service (maybe with Facade) which can be used to register TopicContent classes that can be reused in multiple places (because I'm pretty sure you also added registering content classes to other Topic Resource classess...)